README.md

```diff
@@ -129,10 +151,13 @@ $ make TOOLCHAIN_PREFIX=arm-none-eabi- lk2nd-msmXXXX
   - Arch Linux: `dtc`
 
 Replace `TOOLCHAIN_PREFIX` with the path to your tool chain.
-`lk2nd.img` is built and placed into `build-msm8916-secondary/lk2nd.img`.
+`lk2nd.img` is built and placed into `build-lk2nd-msmXXXX/lk2nd.img`.
+
+**Note:** Unlike lk2nd, lk1st is still experimental and therefore not described
```

Comment:
tbh I'd prefer to not list an experimental feature in the first description paragraphs, maybe move it's description down to here, so it's only lk2nd until there and then "yeah, you can also replace aboot if you know stuff but this is WIP and experimental"?

Reply:
I think lk1st is pretty close and I do say in the introduction that it's for SBCs and expert users only. The only reason I didn't describe it yet is that it needs some careful thought on how to prevent users from copying those commands and destroying their bootloader when they really just want lk2nd.